transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpLoginSecurityFilter.java

```diff
@@ -457,56 +458,69 @@ private void registerCallbacks(
         callbackHandlerMap.put(AuthenticationTokenCallback.class, authenticationTokenCallbackHandler);
 
         String forwarded = httpRequest.getHeader(HEADER_FORWARDED);
```

Comment:
@jitsni Here getHeader will return the first element in the List of Forwarded headers added by intermediaries. Here Forwarded headers are not updated with the current info yet (did not go through proxy handler yet). I am not sure the data in the Forwarded headers is ok to be used, what do you think? Maybe we should just use the session's remote address?